src/main/java/uk/gov/hmcts/reform/civil/service/mediation/MediationCSVService.java

```diff
@@ -69,4 +77,7 @@ protected String getCsvCompanyName(Party party) {
         return (party.isCompany() || party.isOrganisation()) ? party.getPartyName() : null;
     }
 
+    private String isWelshFlag(boolean isBilingualFlag) {
```

Comment:
change this method name. should besome thing getWelshFlagValue/ getBilingualFlag as method starting with is generally return boolean